.github/workflows/run-tests.yaml

```
@@ -0,0 +1,50 @@
+name: Python package
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+        cache: 'pip'
+        cache-dependency-path: setup.py
```

```suggestion
        cache-dependency-path: setup.cfg
```
The setup.py won't change now ?

---

Yes, I think that's the point: If `setup.py` changes, then the cached dependencies are considered stale.

... or idc: We can "optimize less" and let "automatic caching" handle it

---

Nonono - that's "just my fault" 😅 

I didn't know how to write tox `deps`, and since my GH-Action is not using `python3 setup.py test` ... I need to force the deps to get installed somehow